### PR TITLE
Complete approve route point

### PR DESCRIPTION
**Pull request by Ishei5**
- changed RoutePointDto - excluded orders and mapPoint;
 - changed parameter type for reorderRoutePoints method to RoutePoint

### CODE REVIEW

File: goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public RouteActionResponse apply(@PathVariable int id,
     @PreAuthorize("hasAnyRole('ADMIN', 'LOGIST', 'DRIVER')")
     @PutMapping("/{id}/reorder")
     public void reorderRoutePoints(@PathVariable int id,
-                                   @RequestBody LinkedList<RoutePointDto> routePointResponses) {
+                                   @RequestBody LinkedList<RoutePoint> routePointResponses) {
```
Comment (belosh59) [inline goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java:52]: Controller accept persistence Entity?

File: goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package com.goodspartner.web.controller;
 
-import com.goodspartner.web.action.RouteAction;
 import com.goodspartner.dto.RouteDto;
-import com.goodspartner.dto.RoutePointDto;
+import com.goodspartner.entity.RoutePoint;
```
Comment (belosh59) [inline goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java:4]: this mixing of objects should confuse you

File: goods-partner-backend/src/main/java/com/goodspartner/dto/RoutePointDto.java [hunk 3]
```diff
@@ -34,17 +33,4 @@ public class RoutePointDto {
     private LocalTime deliveryStart;
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "HH:mm")
     private LocalTime deliveryEnd;
-
-    private MapPoint mapPoint;
```
Comment (belosh59) [inline goods-partner-backend/src/main/java/com/goodspartner/dto/RoutePointDto.java:38]: here

File: goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java [hunk 4]
```diff
@@ -13,11 +13,13 @@
...
 public interface RouteService {
 
+    RouteDto findById(int routeId);
+
     List<RouteDto> findByDeliveryId(UUID deliveryId);
 
     RouteActionResponse updateRoute(int routeId, RouteAction action);
 
-    void reorderRoutePoints(int id, LinkedList<RoutePointDto> routePointResponses);
+    void reorderRoutePoints(int id, LinkedList<RoutePoint> routePointResponses);
```
Comment (belosh59) [inline goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java:22]: This method should use DTO instead of Entity

**Review by belosh59 — CHANGES_REQUESTED** [review]

File: goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java (hunk 1 above)
Comment (Ishei5) [reply goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java]: Rollback to Dto

File: goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java (hunk 2 above)
Comment (Ishei5) [reply goods-partner-backend/src/main/java/com/goodspartner/web/controller/RouteController.java]: Agree, Rollback to Dto

File: goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java [hunk 5]
```diff
@@ -13,11 +13,13 @@
 
 public interface RouteService {
 
+    RouteDto findById(int routeId);
```
[the remark this answers is not in the document]
Comment (Ishei5) [reply goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java]: Done

File: goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java (hunk 4 above)
Comment (Ishei5) [reply goods-partner-backend/src/main/java/com/goodspartner/service/RouteService.java]: Rollback to Dto

**Review by belosh59 — APPROVED** [review]

**Review by belosh59 — APPROVED** [review]

**Pull request merged**